Dockerfile-ci

```diff
@@ -0,0 +1,23 @@
+#  Copyright 2015 Daniel Giribet <dani - calidos.cat>
```

Comment:
@janisz , yes! Not aware of its existence, and this duplication makes no sense... probably we should also add a link to http://mesos.apache.org/documentation/latest/submitting-a-patch/ in the project README, so all these supporting services are known for new contributors to apache projects. I will add a couple of commits.